### DISCUSSION TITLE
PC ver.: The game crashes when I open that sky mod while I'm in the game.

### DISCUSSION THREAD

**Discussion by gses95193** [thatskymod/Sky-CotL-Scripts#136] [Q&A] [answered]
**The game crashes when I open that sky mod while I'm in the game.
But it worked fine for me before the update**

**I don't know what's wrong, I've tried installing manually, and the game still crash**
![287686](https://github.com/user-attachments/assets/9e8922f0-e249-4f5c-8705-11a831bb7f58)

**Comment by Krullerwhit** [comment, marked as answer]
It's because of the latest update. I'm sure they'll fix it eventually, so just have some patience.

**Comment by thatskymod** [comment]
Hello,

I apologize for the outage. That said, any time the base game is updated, mods also need to be aligned with the new logic. This may take some time.

Consider trying the newly released ThatSkyMod [v3.7.0](https://github.com/XeTrinityz/ThatSkyMod/releases/tag/3.7.0). This version should resolve any outstanding issues found in its predecessors.
